docs/source/model_parameters.md

```diff
@@ -0,0 +1,294 @@
+:::
+
+## Description of model parameters
+
+Schema files describing all simulation model parameters are part of [simtools](https://github.com/gammasim/simtools) and can be found in [simtools/schemas/model_parameters](https://github.com/gammasim/simtools/tree/main/simtools/schemas/model_parameters).
+
+These files describe the simulation model parameters including (among others fields) name, type, format, applicable telescopes, and parameter description.
+They include information about setting and validation activities, data sources, and simulation software.
+The schema files and especially the model parameter descriptions are derived from (and planned to be synchronized with) the [sim_telarray manual](https://www.mpi-hd.mpg.de/hfm/~bernlohr/sim_telarray/).
+
+The schema files are in human readable yaml format and follow a fixed meta metaschema (see [model_parameter_and_data_schema.metaschema.yml](https://github.com/gammasim/simtools/blob/main/simtools/schemas/model_parameter_and_data_schema.metaschema.yml); found in simtools schema directory).
+
+### Example
+
+The following example is for a single parameter description.
+
+```yaml
+%YAML 1.2
+---
+title: Schema for num_gains model parameter
+version: 0.1.0
+meta_schema: simpipe-schema
+meta_schema_url: https://raw.githubusercontent.com/gammasim/simtools/main/simtools/schemas/model_parameter_and_data_schema.metaschema.yml
+meta_schema_version: 0.1.0
+name: num_gains
+description: Number of different gains the input signal gets digitized.
+data:
+  - type: int
+    unit: dimensionless
+    default: 1
+    allowed_range:
+      min: 1
+      max: 2
+instrument:
+  class: Camera
+  type:
+    - LSTN
+    - LSTS
+    - MSTN
+    - MSTS
+    - SSTS
+    - SCTS
+activity:
+  setting:
+    - SetParameterFromExternal
+  validation:
+    - ValidateParameterByExpert
+source:
+  - Initial instrument setup
+simulation_software:
+  - name: sim_telarray
+```
+
+### Valid Keys
+
+Valid keys are described in detail in [model_parameter_and_data_schema.metaschema.yml](https://github.com/gammasim/simtools/blob/main/simtools/schemas/model_parameter_and_data_schema.metaschema.yml). The most important parameters are:
+
+#### Header section
+
+- `title`: Title of the schema file.
+- `version`: Version of this schema file.
+- `meta_schema`: Name of the base schema.
+- `meta_schema_url`: URL of the base schema.
+- `meta_schema_version`: Version of the base schema.
+
+#### Parameter description
+
+- `name`: Name of the parameter.
+- `description`: Description of the parameter.
+- `short_description`: Short description of the parameter (optional).
+
+#### Parameter data
+
+The `data` field is used to describe the actual type, format, and allowed values of the parameter.
+
+- `type`: Data type of the parameter.
+- `unit`: Units of the parameter (compatible with astropy units). Note that units are explicitly listed in the jsonschema.yml file.
```

Comment:
where can I find this jsonschema.yml ? or did you mean the specific schema for this parameter?